grease-cli/src/Grease/Main.hs

```diff
@@ -269,6 +272,35 @@ withMemOptions opts k =
           }
    in k
 
+loadDwarfPreconditions ::
+  ExtShape ext ~ PtrShape ext w =>
+  Mem.HasPtrWidth w =>
+  Symbolic.SymArchConstraints arch =>
+  -- | Is DWARF enabled
+  Bool ->
+  -- | How much to unroll recursive types
+  TypeUnrollingBound ->
```

Comment:
I definitely appreciate the docs - this one is slightly redundant because it is implied by the type itself.